pages.pl/linux/uuidgen.md

```diff
@@ -0,0 +1,17 @@
+# uuidgen
+
+> Stwórz unikalny identyfikator (UUIDs).
+> Zobacz też `uuid`.
+> Więcej informacji: <https://manned.org/uuidgen>.
+
+- Stwórz losowy UUIDv4:
+
+`uuidgen --random`
+
+- Stwórz UUIDv1 oparty o aktualny czas:
+
+`uuidgen --time`
+
+- Stwórz UUIDv5 z nazwy i prefiksu przestrzeni nazw:
+
+`uuidgen --sha1 --namespace {{@dns|@url|@oid|@x500}} --name {{nazwa_obiektu}}`
```

Comment:
The following fix can be applied after the [original](https://github.com/tldr-pages/tldr/blob/main/pages/linux/uuidgen.md) page is updated:

```suggestion
`uuidgen --sha1 --namespace @{{dns|url|oid|x500}} --name {{nazwa_obiektu}}`
```

Common placeholder parts can be moved out of them.